src/Orleans.Core.Abstractions/Runtime/RequestContext.cs

```diff
@@ -142,7 +142,7 @@ public static void Import(Dictionary<string, object> contextData)
             }
         }
 
-        public static Dictionary<string, object> Export(SerializationManager serializationManager)
+        public static Dictionary<string, object> Export(ISerializationManager serializationManager)
```

Comment:
If we extracted the serialization manager interface only for this, I believe it's the wrong approach. Here we were leveraging the DeepCopy method only, but what's most important is that the Import, Export and Clear are all APIs relevant to the runtime, but not to end-users, so they could be split from the abstractions move